app.js

```diff
@@ -2,26 +2,52 @@
 
 const express = require('express');
 const app = express();
+const gapi = require('./domains');
 
 app.set('views', __dirname + '/views');
 app.use(express.static('dist'));
 app.engine('html', require('ejs').renderFile);
 
+let baseEnvironment;
+const config = clientConfiguration();
+
 /**
  * Main view for manual entry
  */
-app.get('*', function (req, res) {
+app.get('/', function (req, res) {
     res.render('main.html', {
-        configuration: clientConfiguration()
+        configuration: config
     });
 });
 
-app.listen(process.env.PORT || 3000, function () {
-    console.log('Listening on port 3000');
+// TODO: Remove - Temporary route to test loggedIn state
+app.get('/loggedIn/:username', function (req, res) {
+    let username = req.params.username;
+    res.render('main.html', {
+        configuration: config,
+        loggedIn: username
+    });
 });
 
+/**
+ * Temporary route for gapi user tasking
+ **/
+app.get('/domains/:username', function(req, res) {
```

Comment:
**A1** + **A2**: Do the temp thing? `/tempfix/domains/:username`. Initially, I had it as `/api/v0` but thought UI shouldn't have an api to begin with.